chat/message/README.md

```diff
@@ -42,16 +42,35 @@ return (
+| `componentOverrides` | `Record<MarkdownComponent, ComponentType>`                             | Uses value to override key'ed markdown elements in terms of how they are rendered |                                                                                                                           |
+| `children`           | `string`                                                               | Rendered children; only string children are supported                             |                                                                                                                           |
+| `isSender`           | `boolean`                                                              | Indicates if the message is from the current user                                 | `true`                                                                                                                    |
+| `markdownProps`      | `LGMarkdownProps`                                                      | Props passed to the internal ReactMarkdown instance                               |                                                                                                                           |
+| `messageRatingProps` | `MessageRatingProps`                                                   | Props to MessageRating component                                                  |                                                                                                                           |
+| `sourceType`         | `'markdown', 'text'`                                                   | Determines the rendering method of the message                                    |                                                                                                                           |
+| `messageBody`        | `string`                                                               | Message body text passed to LGMarkdown                                            |                                                                                                                           |
+| `verified`           | `{ verifier?: string; verifiedAt?: Date; learnMoreUrl?: string; }`     | Sets if an answer is "verified" and controls the content of the message banner.   |                                                                                                                           |
+| `links`              | `{ url: string; text: string; imageUrl?: string; variant: string; }[]` | A list of links to show in a section at the end of the message.                   |                                                                                                                           |
```

Comment:
I'm starting to think that maybe these would be better as a slot where you pass the `<RichLink>` component instead of an object. I believe you mentioned on another comment that consumers could potentially use `RichLink`'s independently. If that's the case, then we should keep the usage of `RichLink` consistent. And this way, it would be easier to swap `RichLink`'s between different components. Thoughts?

Reply:
I like the idea of component composition in general. Here though I think it's a bit tricky because we'd want to enforce that you're passing `RichLink` components to this slot. Consumers would also need to wrap all of the rich links they pass in a fragment which is a little more friction.

The way I think of it (as set up here) the `links` prop is data that gets passed to the `MessageLinks` component. If you want a custom look for this slot you'd define `componentOverrides.MessageLinks`.

The case for independent rich links is foggy future but would be something like rendering links in the message markdown as rich links, or creating a custom component that also uses rich links.